wordpress-mobile/WordPress-Android
# Add log persistence

**To test:**
- Ensure CI passes – this PR contains test coverage for all of its functionality.
- Run WPAndroid from Android Studio – sign into WP, and go through a few screens. Close the app.
- In Android Studio, under Device File Explorer, navigate to `data > user > 0 > org.wordpress.android > logs. View the content of the most recent log file and note that it contains appropriate information based on your actions.
- Stop and start the app four more times, and note that your first log file has been erased – this is to ensure we don't use an inordinate amount of the user's storage. In the future, this could be a user setting that can be customized when trying to debug a problem.
- Launch the app and ensure the existing logging functionality is the same as it was before – it's under Me > Help & Support > Application log.

**PR submission checklist:**
